web/profiles/custom/yalesites_profile/config/sync/block.block.alertblock.yml

```diff
@@ -1,21 +0,0 @@
-uuid: a0ac4316-3e39-4a46-a8d1-edb87feb0d52
```

Comment:
Should this have been deleted? Looks like we lost a block along the way.

Reply:
Not sure... I just did a config import, and I have it, but its machine name is `alert_block` (with an underscore) so maybe we had two before?